addons/l10n_id_efaktur_coretax/models/account_move.py

```diff
@@ -0,0 +1,317 @@
+        ('TD.01122', '22 - BKP strategis tertentu dalam bentuk anode slime dan emas butiran'),
+        ('TD.01123', '23 - untuk penyerahan kertas koran dan/atau majalah'),
+        ('TD.01124', '24 - PPN tidak dipungut oleh Pemerintah lainnya'),
+        ('TD.01125', '25 - BKP dan JKP tertentu'),
+        ('TD.01126', '26 - Penyerahan BKP dan JKP di Ibu Kota Negara baru'),
+        ('TD.01127', '27 - Penyerahan kendaraan listrik berbasis baterai'),
+    ])
+
+    # Extra selection after choosing l10n_id_kode_transaksi 08
+    l10n_id_add_info_08 = fields.Selection([
+        ('TD.00501', '1 - PPN Dibebaskan Sesuai PP Nomor 146 Tahun 2000 Sebagaimana Telah Diubah Dengan PP Nomor 38 Tahun 2003'),
+        ('TD.00502', '2 - PPN Dibebaskan Sesuai PP Nomor 12 Tahun 2001 Sebagaimana Telah Beberapa Kali Diubah Terakhir Dengan PP Nomor 31 Tahun 2007'),
+        ('TD.00503', '3 - PPN dibebaskan berdasarkan Peraturan Pemerintah Nomor 28 Tahun 2009'),
+        ('TD.00504', '4 - (Tidak ada cap)'),
+        ('TD.00505', '5 - PPN Dibebaskan Sesuai Dengan PP Nomor 81 Tahun 2015'),
+        ('TD.00506', '6 - PPN Dibebaskan Berdasarkan PP Nomor 74 Tahun 2015'),
+        ('TD.00507', '7 - (tanpa cap)'),
+        ('TD.00508', '8 - PPN DIBEBASKAN SESUAI PP NOMOR 81 TAHUN 2015 SEBAGAIMANA TELAH DIUBAH DENGAN PP 48 TAHUN 2020'),
+        ('TD.00509', '9 - PPN DIBEBASKAN BERDASARKAN PP NOMOR 47 TAHUN 2020'),
+        ('TD.00510', '10 - PPN Dibebaskan berdasarkan PP Nomor 49 Tahun 2022'),
+    ])
+
+    l10n_id_facility_info_08 = fields.Selection([
+        ('TD.01101', '1 - PPN Dibebaskan Sesuai PP Nomor 146 Tahun 2000 Sebagaimana Telah Diubah Dengan PP Nomor 38 Tahun 2003'),
+        ('TD.01102', '2 - PPN Dibebaskan Sesuai PP Nomor 12 Tahun 2001 Sebagaimana Telah Beberapa Kali Diubah Terakhir Dengan PP Nomor 31 Tahun 2007'),
+        ('TD.01103', '3 - PPN dibebaskan berdasarkan Peraturan Pemerintah Nomor 28 Tahun 2009'),
+        ('TD.01104', '4 - (Tidak ada cap)'),
+        ('TD.01105', '5 - PPN Dibebaskan Sesuai Dengan PP Nomor 81 Tahun 2015'),
+        ('TD.01106', '6 - PPN Dibebaskan Berdasarkan PP Nomor 74 Tahun 2015'),
+        ('TD.01107', '7 - (tanpa cap)'),
+        ('TD.01108', '8 - PPN DIBEBASKAN SESUAI PP NOMOR 81 TAHUN 2015 SEBAGAIMANA TELAH DIUBAH DENGAN PP 48 TAHUN 2020'),
+        ('TD.01109', '9 - PPN DIBEBASKAN BERDASARKAN PP NOMOR 47 TAHUN 2020'),
+        ('TD.01110', '10 - PPN Dibebaskan berdasarkan PP Nomor 49 Tahun 2022'),
+    ])
+
+    l10n_id_kode_transaksi = fields.Selection(selection_add=[('10', '10 Penyerahan lainnya')])
+    l10n_id_add_info = fields.Char()
+    l10n_id_stlg_rate = fields.Float(string="STLG Rate", default=0, copy=False)  # luxury good rate
+    l10n_id_efaktur_available = fields.Boolean(compute="_compute_l10n_id_efaktur_available")
+
+    l10n_id_efaktur_document = fields.Many2one('l10n_id_efaktur_coretax.document', readonly=True, copy=False, string="e-Faktur Document")
```

Comment:
Be careful that in 18+ you will have the newer version of the efaktur module. You will have issues if your fields have the same name.
You could prefix every fields with l10n_id_efaktur_coretax (or l10n_id_coretax to stay short)

Reply:
since other fields are not there in 18.0, I'll change just the `l10n_id_efaktur_document` field for now then

Reply:
I think it's best to not need to think about it and just use the same prefix for all fields :ok_hand: 
It's only a technical choice but it won't hurt and can avoid mistakes

Reply:
I see, will do that then